diff --git a/asar-contents/app/main.js b/asar-contents/app/main.js
index 627a1b5..2fa0cd0 100644
--- a/asar-contents/app/main.js
+++ b/asar-contents/app/main.js
@@ -501,7 +501,7 @@ async function createWindow() {
   const width = windowConfig ? Math.min(windowConfig.width, maxWidth) : DEFAULT_WIDTH;
   const height = windowConfig ? Math.min(windowConfig.height, maxHeight) : DEFAULT_HEIGHT;
   const windowOptions = {
-    show: false,
+    show: true,
     width,
     height,
     minWidth: MIN_WIDTH,
