diff --git a/app/main.main.ts b/app/main.main.ts
index d9016c7..8d490af 100644
--- a/app/main.main.ts
+++ b/app/main.main.ts
@@ -704,7 +704,7 @@ async function createWindow() {
     !app.getLoginItemSettings().wasOpenedAsHidden && !startInTray;
 
   const windowOptions: Electron.BrowserWindowConstructorOptions = {
-    show: false,
+    show: true,
     width,
     height,
     minWidth: MIN_WIDTH,
